testSample/features/par2.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Logging two
```

Comment:
Pls consider to add tests for:
Attachments (png, json, txt) send to RP as log message
Send screenshots as log message
Run scenarios with 'Scenario Outlines'
Run scenarios where steps receive Tables
Run scenarios with 'Example' part
Run scenario where step is failed, skipped, undefined